gitlab/v4/objects/users.py

```diff
@@ -283,6 +283,48 @@ def activate(self, **kwargs: Any) -> Union[Dict[str, Any], requests.Response]:
             self._attrs["state"] = "active"
         return server_data
 
+    @cli.register_custom_action("User")
+    @exc.on_http_error(exc.GitlabBanError)
+    def ban(self, **kwargs: Any) -> Union[Dict[str, Any], requests.Response]:
+        """Ban the user.
+
+        Args:
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabBanError: If the user could not be banned
+
+        Returns:
+            Whether the user has been banned
```

Comment:
I'm thinking that this wording is not entirely clear but this is what we have in the older ones so we can change that consistently in a follow-up :)